test/integration/test-user-error-reporting.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+    withdrawRequest,
+    depositRequestUrl,
+} from '../../testing/test-helper';
+
+describe('user-error-reporting', function() {
+  this.timeout(15000);
+  const randomId = Math.random();
+
+  describes.integration('user-error-reporting integration test', {
+    extensions: ['amp-analytics'],
+    hash: 'log=0',
+    body: `   
+    <amp-analytics><script type="application/json">
+          {
+              "requests": {
+                  "user-error": "${depositRequestUrl(randomId)}"
+              },
+              "triggers": {
+                  "user-error": {
+                      "on": "user-error",
+                      "request": "user-error"
+                  }
+              }
+          }
+      </script></amp-analytics>
+
+      <amp-pixel src="https://foo.com/tracker/foo"
+               referrerpolicy="fail-referrer">`,
+  }, env => {
+    it('should ping correct host with amp-pixel referrer error', () => {
+      return withdrawRequest(env.win, randomId).then(request => {
+        expect(request).to.be.ok;
+      });
+    });
+  });
+
+  describes.integration('user-error-reporting integration test', {
+    extensions: ['amp-analytics'],
+    hash: 'log=0',
+    /* eslint-disable */
+    body: `
+    <amp-img
+      src="../../examples/img/sea@1x.jpg"
+      srcset="../../examples/img/sea@1x.jpg 1x, ../../examples/img/sea@2x.jpg 2x"
+      alt="Fusce pretium tempor justo, vitae consequat dolor maximus eget."
+      placeholder
+      width="360" height="216" layout="responsive"
+      role='img'>
+    </amp-img>
+    
+    <amp-analytics><script type="application/json">
+          {
+              "requests": {
+                  "user-error": "${depositRequestUrl(randomId)}"
+              },
+              "triggers": {
+                  "user-error": {
+                      "on": "user-error",
+                      "request": "user-error"
+                  }
+              }
+          }
+      </script></amp-analytics>`,
+  }, env => {
+    it('should ping correct host with amp-img error', () => {
```

Comment:
please indicate that we're testing `user().assert` and `user().error()` in this 2 test cases